.swiftlint.yml

```diff
@@ -0,0 +1,155 @@
+opt_in_rules:
+- attributes
+- closure_end_indentation
+- closure_spacing
+- empty_count
+- explicit_init
+- fatal_error_message
+- file_header
+- first_where
+- implicit_return
+- implicitly_unwrapped_optional
+- nimble_operator
+- no_extension_access_modifier
+- number_separator
+- operator_usage_whitespace
+- overridden_super_call
+- private_outlet
+- prohibited_super_call
+- redundant_nil_coalescing
+- switch_case_on_newline
+
+included:
+- Source
+- Tests
+
+excluded:
+- Carthage
+
+line_length: 160
+
+file_header:
+  required_pattern: |
+                    \/\/
+                    \/\/  .*?\.swift
+                    \/\/  Carthage
+                    \/\/
+                    \/\/  Created by [\S ]+ on \d{1,2}\/\d{1,2}\/\d{2}\.
+                    \/\/  Copyright © \d{4} Carthage\. All rights reserved\.
+                    \/\/
```

Comment:
Sure about that? Shall I make this rule ensure we don't have a file header then?